cinema/serializers.py

```diff
@@ -125,9 +125,9 @@ class TicketSerializer(serializers.ModelSerializer):
     def validate(self, attrs):
         data = super(TicketSerializer, self).validate(attrs=attrs)
```

Comment:
In the `validate` method, using `attrs` directly might not be the best practice. Consider using `validated_data` to ensure you're working with already validated data.